xapian-core/tests/api_weight.cc

```diff
@@ -596,6 +596,22 @@ DEFINE_TESTCASE(tfidfweight3, backend) {
     mset_expect_order(mset2, 2, 4);
     TEST_EQUAL_DOUBLE(15 * mset[0].get_weight(), mset2[0].get_weight());
 
+    //check for "nfn" when termfreq != N
```

Comment:
Just a few style nit-picks left - please put a space between `//` and the comment text for readability (also below).